tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -18,38 +18,38 @@
 
     <!-- Integration: AWSSDK.Core -->
     <!--    Assembly: AWSSDK.Core -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Core/3.7.400.30 -->
-    <PackageReference Include="AWSSDK.Core" Version="3.7.400.30" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Core/3.7.400.33 -->
+    <PackageReference Include="AWSSDK.Core" Version="3.7.400.33" />
 
     <!-- Integration: AWSSDK.DynamoDBv2 -->
     <!--    Assembly: AWSSDK.DynamoDBv2 -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.DynamoDBv2/3.7.401.8 -->
-    <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.401.8" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.DynamoDBv2/3.7.401.11 -->
+    <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.401.11" />
 
     <!-- Integration: AWSSDK.Kinesis -->
     <!--    Assembly: AWSSDK.Kinesis -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.402.7 -->
-    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.402.7" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Kinesis/3.7.402.10 -->
+    <PackageReference Include="AWSSDK.Kinesis" Version="3.7.402.10" />
 
     <!-- Integration: AWSSDK.SimpleNotificationService -->
     <!--    Assembly: AWSSDK.SimpleNotificationService -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.30 -->
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.30" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.400.33 -->
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.400.33" />
 
     <!-- Integration: AWSSDK.SQS -->
     <!--    Assembly: AWSSDK.SQS -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.30 -->
-    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.30" />
+    <!-- Latest package https://www.nuget.org/packages/AWSSDK.SQS/3.7.400.33 -->
+    <PackageReference Include="AWSSDK.SQS" Version="3.7.400.33" />
 
     <!-- Integration: Azure.Messaging.ServiceBus -->
     <!--    Assembly: Azure.Messaging.ServiceBus -->
-    <!-- Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.1 -->
+    <!-- Latest package https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.18.2 -->
     <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.17.5" />
```

Comment:
🤔 Hmmm, we should probably test and bump this soon seeing as we're at a different minor version